### DISCUSSION TITLE
AD Block defaulting to choose a blocklist provider

### DISCUSSION THREAD

**Discussion by GUGAP3000** [RaspAP/raspap-webgui#971] [Q&A]
I'm trying to enable either notracking-hostnames or notracking-domains inside the blocklist settings. The problem that I'm facing is that every time I try to save the settings or restart Ad Blocking it defaults back to choose a blocklist provider.

![Capture11](https://user-images.githubusercontent.com/87623945/127234176-7e821600-0d2c-4903-ac98-6a6f4927b551.JPG)
 
When pressing "Save settings" or Restart Ad Blocking" it goes back to choose a blocklist provider

![Capture12](https://user-images.githubusercontent.com/87623945/127234267-2adeebff-9f40-40d5-baf0-14d84aa15ce8.JPG)

**Additional context**
Hi, thanks for the amazing code! How should I procced in order to enable ad block? Thanks again!

**Comment by billz** [comment]
Adblock is already enabled when you choose it during the install. The blocklist provider option is used for updating the lists, which you should do periodically. 

More here https://docs.raspap.com/adblock/
